### ISSUE TITLE
swipeable-pages will not start the tracking when element have a left position > 0

### ISSUE THREAD

**Issue by dartpast** [MeTaNoV/iron-swipeable-pages#44]
Swipeable-pages will not start the tracking when the left position > 0, due to incorrect track.x value in _onTrack function. The x-value have to be relative to the elements position otherwise the edgeSwipeSensitivity-check will not accept the value.

```
<app-header-layout>
  <app-header>
    <app-toolbar>Test iron-swipeable-pages</app-toolbar>
  </app-header>
  <div class="layout horizontal">
    <div class="flex">Left column</div>
    <iron-swipeable-pages>
      <div>Page 1</div>
      <div>Page 2</div>
    </iron-swipeable-pages>
  </div>
</app-header-layout>
```

With a window width of 1000px the track.x value will be 500 or greater, and the this.offsetWidth value 500 making
`(track.x - track.dx) > this.offsetWidth - this.edgeSwipeSensitivity`
true most of the times.

Adding the line: 
`track.x = track.x - this.getBoundingClientRect().left;`
after
`var track = event.detail;`
in the _onTrack function made the trick for me.

**Comment by MeTaNoV** [comment]
Hi @dartpast ,
thanks for spotting the bug! Are you eager to submit your solution as a PR, or do you prefer that I do it?
Cheers,
Pascal

**Comment by dartpast** [comment]
@MeTaNoV 
You can include it to your next release :)

**Comment by MeTaNoV** [comment]
@dartpast  should be fixed by release 1.2.7? Can you confirm and re-open if not?
